src/screens/AccountRecover.js

```diff
@@ -22,7 +22,7 @@ import {
   findNodeHandle,
```

Comment:
when entering a recovery phrase, the field shows green as long as the words are bip39, and it'll let me continue to create pin and confirm, but crashes if the entire prhase was not actually valid. 
<img width="357" alt="Screenshot 2019-08-20 at 16 13 43" src="https://user-images.githubusercontent.com/10432070/63356116-890eb100-c367-11e9-94af-acf583d58353.png">

Reply:
should be solved by the last commit.